libraries/AP_HAL_ChibiOS/hwdef/CSKY405/Readme.md

```diff
@@ -0,0 +1,101 @@
+# CSKY405 Flight Controller
+
+The CSKY405 is a flight controller produced by [ClearSky](http://csky.space/).
+
+## Features
+    Processor
+        STM32F405 168Mhz, 1MB 32-bit processor
+        AT7456E OSD
+    Sensors
+        BMI088 Acc/Gyro
+        BMP390 barometer
+    Power
+        2S - 6S Lipo input voltage with voltage monitoring
+        90A Cont., 215A peak current monitor
+        12V, 2A BEC for powering Video Transmitter controlled by GPIO
+        5V, 2.5A BEC for internal and peripherals
+    Interfaces
+        10x PWM outputs DShot capable
+        1x RC input
+        6x UARTs/serial for GPS and other peripherals
+        I2C port for external compass, airspeed, etc.
+        microSDCard for logging, etc.
+        USB-C port
+  
+
+## Overview
+
+![CSKY405](CSKY405.png)
+
+## Wiring Diagram
+
+![CSKY405 Wiring](CSKY405_wiring.png)
+
+## UART Mapping
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+
+ - SERIAL0 -> USB
+ - SERIAL1 -> UART4  (MAVLink2 telem)
+ - SERIAL2 -> USART1 (Serial RC input) (DMA capable)
+ - SERIAL3 -> UART5  (GPS)
+ - SERIAL4 -> USART3 (User) (TX DMA capable)
+ - SERIAL5 -> USART6 (User) (TX DMA capable)
+ - SERIAL6 -> USART2 (RX tied to inverted SBUS RC input, but can be used as normal UART if :ref:`BRD_ALT_CONFIG<>` =1) 
```

Comment:
```suggestion
 - SERIAL6 -> USART2(XMA Capable) (RX tied to inverted SBUS RC input, but can be used as normal UART if :ref:`BRD_ALT_CONFIG<>` =1) 
```
I still dont see a separate SBUS PIN ONLY RX2input...ehere is it? mut be separate for this alt config use to work

Reply:
It really doesn't exist on the board, there is only RX2, but in the current configuration the SBUS works. Did I understand correctly that since I do not have a logic inverter on the board, I will not be able to configure this USART as a regular one? Only SBUS?

Reply:
since RC2RX2 is an rc interrupt normally, and works for all single wire rc protocols.....since RX2 is direct to the uart without an intervening inverter, the alt config will work for CRSF,etc. and should use uart2 since it has dma and uart4 does not...it wont work well for crsf because of that..i modified my suggested change above since there is no inverter